ci-operator/step-registry/rosa/lifecycle/rosa-lifecycle-workflow.yaml

```diff
@@ -4,6 +4,8 @@ workflow:
     pre:
     - ref: rosa-setup
     - ref: rosa-setup-readiness-cluster
+    - ref: rosa-setup-conf-external-auth
+    - ref: rosa-setup-conf-cilium
```

Comment:
I do not think adding an extraordinary step to the typical workflow is an excellent way. A better way to do this is to start a new workflow. The workflow is a fantastic way to help us with different configurations; it is complex and unreadable to combine all of the logic into one.